packages/webdriverio/src/middlewares.js

```diff
@@ -36,6 +36,12 @@ export const elementErrorHandler = (fn) => (commandName, commandFn) => {
 
                     return await fn(commandName, commandFn).apply(this, args)
                 }
+                if (typeof error === CustomRequestError) {
+                    throw new Error(
```

Comment:
@CrispusDH One small suggestion. When using backticks it'll preserve the newlines so you don't need to concatenate or specify a newline.

```suggestion
throw new Error(`
    Can't call ${commandName} on element with selector "${this.selector}" because:
    ${error}
`)
```

Reply:
Thanks :)